roles/brute-ora-cleanup/tasks/main.yml

```diff
@@ -456,6 +456,46 @@
   become_user: root
   shell: awk '/^oracle/ {print $1}' /proc/modules | sudo xargs -r modprobe -r
 
+- name: Reset kernel sysctl values to EL defaults
+  sysctl:
+    name: "{{ item.parameter }}"
+    value: "{{ item.value }}"
+    state: present
+    sysctl_set: true
```

Comment:
In a perfect world, we'd set the parameters to what we think are Red Hat's defaults on the live system, but remove entirely from /etc/sysctl.conf.  But given that the ANsible sysctl module doesn't have an option for that, what's here should be fine.